controller/src/main/java/org/jboss/as/controller/transform/description/ResourceTransformationDescriptionBuilder.java

```diff
@@ -86,8 +114,13 @@ public interface ResourceTransformationDescriptionBuilder extends Transformation
      *
      * @param definition the resource definition
      * @return the builder for the child resource
+     * @deprecated Use {@link ResourceTransformationDescriptionBuilder#addChildResource(ResourceRegistration)} instead.
      */
-    ResourceTransformationDescriptionBuilder addChildResource(ResourceDefinition definition);
+    @Deprecated(forRemoval = true)
```

Comment:
Note to reviewers: I only see 5 uses of this in WF, all in the legacy Picketbox extensions.